### PR TITLE
Rabbit deaths

### PR DESCRIPTION
**Pull request by eviedemetriou**
Contains updated death method and its test

### CODE REVIEW

File: rabbit_project/classes/rabbit_death.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Rabbits:
```
Comment (davidrichardharvey) [inline rabbit_project/classes/rabbit_death.py]: Why is this file here?

**Review by CharlotteKings — CHANGES_REQUESTED** [review]
Remove rabbit_death.py

**Review by CharlotteKings — APPROVED** [review]
